src/libsystemd/sd-journal/journal-file.c

```diff
@@ -92,6 +92,19 @@
 #  pragma GCC diagnostic ignored "-Waddress-of-packed-member"
 #endif
 
+_const_ static int mmap_prot_from_open_flags(int flags) {
```

Comment:
`_const_` has no relevance for functions that are static anyway. `_const_` is a hint to the compiler that some function doesn't need to be called twice for the same value because it is side-effect free. The compiler can't know that if it looks at a public symbol, so we can tell it that. But, if the function is "static", then the compiler knows anyway if is const or not.

(Moreover, given you added assert_not_reached() here, it's not side-effect free anymore anyway, so it's not const either anymore)

Reply:
I dropped the `_const_` now.